src/main/java/seedu/address/logic/commands/DeleteStudentCommand.java

```diff
@@ -19,8 +19,8 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Deletes the student identified by the index number used in the displayed student list.\n"
-            + "Parameters: INDEX (must be a positive integer)\n"
-            + "Example: " + COMMAND_WORD + " 1";
+            + "\nParameters: INDEX (must be a positive integer)"
```

Comment:
Hi, just want to check as the previous line ": Deletes the student identified by the index number used in the displayed student list.\n" already has an \n, does having an additional \nParameters here cause anything weird on the UI? Might be better to just keep one.

Reply:
Yes you are right, I will resolve in the next commit!